cypress/integration/translations.cy.js

```diff
@@ -1,7 +1,7 @@
 import { EXTENDED_TIMEOUT } from '../support/util.js'
 
-const UPDATE_BUTTON_ORIGINAL = 'Update'
-const UPDATE_BUTTON_TRANSLATED = 'Oppdater'
+const UPDATE_ITEM_ORIGINAL = 'Update'
```

Comment:
Why "Item", it's referring to a button 🤷 
Just like "WELCOME_MSG" is referring to the message...

Reply:
Unintentional, replace all mistake probably.